src/toast/ops/mapmaker.py

```diff
@@ -221,9 +221,10 @@ def _exec(self, data, detectors=None, **kwargs):
             raise RuntimeError("You must set the det_data trait before calling exec()")
 
         # Check map binning
+        map_binning = self.map_binning
```

Comment:
A note on this change:  I am doing this with a local variable so that the original value of the `self.map_binning` trait is preserved instead of being overwritten.